### PR TITLE
Client files

### PR DESCRIPTION
**Pull request by sujithavikram**

### CODE REVIEW

**Comment by sujithavikram** [comment]
@jennqiao @2000prcs @leealbert95 please review my pull request

File: client/src/components/Paginations.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    constructor(props) {
+        super(props);
+        this.state = {
+
+        };
+        this.handleClick = this.handleClick.bind(this);
+        this.renderPageIndex = this.renderPageIndex.bind(this);
+    }
+    handleClick(i) {
+        console.log('clicked-', i);
```
Comment (jennqiao) [inline client/src/components/Paginations.jsx]: no more need for console logs :)

File: client/src/components/Review.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    componentDidMount() {
+        this.getReviews();
+    }
+    componentDidUpdate(prevProps) {
+        if (prevProps.locationId !== this.props.locationId) {
+            this.getReviews();
+        }
+    }
+    getReviews(pageIndex, keyword){
+        if (pageIndex == null) {   //checks for both null and undefined
```
Comment (jennqiao) [inline client/src/components/Review.jsx:33]: you can simplify the line here by setting
pageIndex = pageIndex || this.state.pageIndex

Or the es6 way, you can set pageIndex to be default this.state.pageIndex in the parameters (line 32) ie pageIndex = this.state.pageIndex.

And same with keyword == null

File: client/src/components/Review.jsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        }
+        axios.get(`http://127.0.0.1:3000/reviews/`, {
+            params : {
+                locationId: this.props.locationId,
+                index: pageIndex,
+                keyword: keyword
+            }
+        })
+        .then(reviews => { 
+            console.log('axios get req reviews-', reviews);
```
Comment (jennqiao) [inline client/src/components/Review.jsx]: you can also get rid of console logs here too :)

**Review by jennqiao — COMMENTED** [review]
Looks good! just left a few comments.

File: client/src/components/Paginations.jsx (hunk 1 above)
Comment (sujithavikram) [reply client/src/components/Paginations.jsx]: I'll remove it

File: client/src/components/Review.jsx (hunk 2 above)
Comment (sujithavikram) [reply client/src/components/Review.jsx:33]: I'm particularly checking for null and undefined condition here. If I make it pageIndex = pageIndex || this.state.pageIndex, an empty string will be evaluated to false which I'm not looking for..

**Pull request merged**